src/open_inwoner/templates/pages/contactform/form.html

```diff
@@ -1,9 +1,9 @@
 {% extends 'master.html' %}
-{% load i18n contact_tags %}
+{% load i18n solo_tags %}
 
 {% block content %}
     <h1 class="h1">
         {% trans "Contactformulier" %}
     </h1>
-{% contact_form form_object=form %}
+    {% include "components/Contact/ContactForm.html" with form_object=form has_form_configuration=has_form_configuration %}
```

Comment:
Passing `csrf_token=csrf_token` when using `only` should work, tested this locally as well (https://stackoverflow.com/a/44419370). I'd prefer that over dropping `only`